lib/authc/AssertionAuthenticationResult.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
```

Comment:
Any special reason why this file is here and not in the `lib/resource/` directory, just like `AuthenticationResult.js`?